.github/workflows/test-cli.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: "Test det CLI"
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'docs/*'
+  push:
+    branches:
+      - 'main'
+      - 'releases/**'
+
+jobs:
+  test-cli:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
+        os: ["ubuntu-latest", "windows-latest"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install det dependencies
+        run: |
+          python --version
+          python -m pip install --upgrade pip
+          pip --version
+          pip install wheel cryptography patch-ng
```

Comment:
there used to be context as to why these two lines were here... maybe keep the comment that explained them?

If you want to install the extra deps on linux too, that's fine.  But leave the explanations please.